ftw/upgrade/tests/test_upgrade_step_intids.py

```diff
@@ -0,0 +1,7 @@
+from ftw.upgrade.testing import INTID_UPGRADE_FUNCTIONAL_TESTING
+from ftw.upgrade.tests.test_upgrade_step import TestUpgradeStep
+
+
+class TestUpgradeStepIntids(TestUpgradeStep):
```

Comment:
Important is that the code runs and does not break.
I don't think we could test that it actually does the job, since we'd have to stub the ZODB caching etc, which is not something we want to do.
But you did manually run tests to verify that your code helps?